### PR TITLE
Update documentation for 1.17.0

### PR DESCRIPTION
**Pull request by ccedric**
Modifications:
- We now support cassandra 4
- Updated Print Pdf section
- Updated Gce page
- small typos fixed

### CODE REVIEW

File: content/reference/current/installation/docker/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Make sure you are logged in using `docker login` with the username you provided
 
 ### Preparing the Cassandra database
 
-We will be using the official Cassandra image under the tag name `cassandra:3`.
+We will be using the official Cassandra image under the tag name `cassandra:4`.
```
Comment (slandelle) [inline content/reference/current/installation/docker/index.md:61]: If we do that, we must also:
* update the installer to use Cassandra 4
* document the migration path

**Review by slandelle — CHANGES_REQUESTED** [review]

**Review by Isammoc — APPROVED** [review]

File: content/reference/current/installation/manual/index.md [hunk 2]
```diff
@@ -52,6 +52,13 @@ If using systemd, do a `systemctl start cassandra`. It is also possible to start
 
 Go to the installation folder, and perform the following command: `./bin/frontline`. More details in the [launch section]({{< ref "#launch-gatling-enterprise" >}}).
 
+### Upgrading Cassandra 3 to Cassandra 4
+
+If you are currently using Cassandra 3 with your Gatling Enterprise, you may want to upgrade to the latest version of Cassandra 4.
+The upgrade is pretty simple, keep the same `cassandra.yaml` and modify the following configuration:
+
+`num_tokens: 16`
```
Comment (notdryft) [inline content/reference/current/installation/manual/index.md:60]:
```suggestion
```yaml
num_tokens: 16
```
```

File: content/reference/current/installation/manual/index.md [hunk 3]
```diff
@@ -65,8 +72,7 @@ We recommend you use JDK builds from [Azul System's Zulu](https://www.azul.com/d
 Other JVMs such as OpenJ9 are not supported.
 
 {{< alert warning >}}
-- As of Cassandra 3, JDK 11 support is flagged as experimental.
-- As a consequence, we only support running Cassandra with JDK 8.
+- Cassandra 3 supports only JDK 8, while Cassandra 4 support JDK 8 &  11.
```
Comment (notdryft) [inline content/reference/current/installation/manual/index.md:75]:
```suggestion
- Cassandra 3 supports only JDK 8, while Cassandra 4 supports JDK 8 & 11.
```

File: content/reference/current/user/reports/index.md [hunk 4]
```diff
@@ -121,9 +121,9 @@ Choose an expiration date, then click the generate button.
 
 You can copy the public link to share your report to non-Gatling Enterprise users, or click on the "Go" Button to access it yourself. You can click on the "OK" button to close this modal.
 
-## Export PDF
+## Print PDF
```
Comment (notdryft) [inline content/reference/current/user/reports/index.md:124]:
```suggestion
## Print to PDF
```

**Pull request merged**